### PR TITLE
vapoursynth: Fix loading python module

### PR DESCRIPTION
**Pull request by Biswa96**
Fixes https://github.com/msys2/MINGW-packages/issues/10528 and Fixes https://github.com/msys2/MINGW-packages/issues/6630. Please review.

### CODE REVIEW

File: mingw-w64-vapoursynth/PKGBUILD [hunk 1]
```diff
@@ -58,4 +61,6 @@ package() {
 
   rm -f ${pkgdir}${MINGW_PREFIX}/lib/vapoursynth/*.dll.a
   rm -f ${pkgdir}${MINGW_PREFIX}/lib/python${_py3ver}/site-packages/*.dll.a
+  cp ${pkgdir}${MINGW_PREFIX}/lib/python${_py3ver}/site-packages/vapoursynth.dll \
+     ${pkgdir}${MINGW_PREFIX}/lib/python${_py3ver}/site-packages/vapoursynth.pyd
```
Comment (naveen521kk) [inline mingw-w64-vapoursynth/PKGBUILD:65]: Isn't there a better fix than this?

Comment (naveen521kk) [reply mingw-w64-vapoursynth/PKGBUILD:65]: I mean there should be something in the build system itself which reads the suffix from sysconfig/distutils, right?

Comment (raedrizqie) [reply mingw-w64-vapoursynth/PKGBUILD:65]: yes.. setup.py must be patched

Comment (Biswa96) [reply mingw-w64-vapoursynth/PKGBUILD:71]: OK, I am merging this for the vspipe issue. Feel free to create another pull request for the setup.py.

**Pull request merged**